Gruntfile.js

```diff
@@ -175,8 +162,8 @@ module.exports = (grunt) => {
   grunt.loadNpmTasks('grunt-webpack');
 
   // Default task(s).
-  grunt.registerTask('default', ['eslint', 'webpack:buildDev', 'webpack:build', 'concat', 'docco']);
-  grunt.registerTask('test', 'Run qunit tests.', ['webpack:buildDev', 'concat', 'qunit']);
+  grunt.registerTask('default', ['clean', 'eslint', 'webpack:buildDev', 'webpack:build', 'concat', 'docco']);
+  grunt.registerTask('test', 'Run qunit tests.', ['clean', 'webpack:buildDev', 'concat', 'qunit']);
```

Comment:
**clean** does not delete the releases folder, it deletes the build folder. With it we make sure that if the process is not properly finished, there are no remaining files from a previous build. I would not test / build on a live webpage there are many errors in the process that can impact on the availability.
This is just a suggestion anyway, it can be removed.

Reply:
I think clean makes sense here. This is how you would do in java with maven: `mvn clean package`, which also removes target folder.